browser_history/generic.py

```diff
@@ -377,3 +416,10 @@ def default(self, o):
             )
 
         return json_string
+
+    format_map: Dict[str, Any] = {
+        'csv': to_csv,
+        'json': to_json,
+        'jsonl': partial(to_json, json_lines=True)
+    }
+    """Dictionary which maps output formats to their respective functions."""
```

Comment:
Any particular reason this is at the end and not alongside `field_map`?

Reply:
because to_json and to_csv is not yet defined until this line.